NitroxClient/MonoBehaviours/MultiplayerMovementController.cs

```diff
@@ -0,0 +1,174 @@
+using System;
+using System.Collections.Generic;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxModel.Core;
+using NitroxModel.DataStructures;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours;
+
+public class MultiplayerMovementController : MonoBehaviour
+{
+    private static readonly Dictionary<NitroxId, MultiplayerMovementController> movementControllersById = new();
+
+    public float TimeScalar { get; set; } = 1f;
```

Comment:
Can this be a const?

Reply:
I suppose, though I figured there may be some instances we could change the TimeScalar say for example when Subnautica's TimeScalar changes